ui/src/message_popup/author/new_question_page.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint no-console: "off" */
```

Comment:
Because the function to create a new question was moved here and it logs what its doing to the console, there was a need for the eslint suppression. However, I'm pretty sure by now from what it has already logged to the console in the past that the functions are working properly so I can remove the logging entirely.